### PR TITLE
Fix more UX/design debt

### LINKED ISSUE #101: UX/design debt 2
- [x] On /input the link 'Bekijk de lijst met alle stembureaus' should have cursor:pointer (applies to all links)
- [x] The 'Shift + Enter' text should not break over multiple lines. Use `white-space: nowrap;`. Add some padding/margin below the 'Beginnen' button so the 'shift + enter' text is positioned neatly below the button (screenshot of how it should look: .5rem between button and text, text has .5rem margin-left (iso 2rem) Scherm­afbeelding 2024-06-26 om 08 27 17
- [x] There should be a spacer before the input for the list-totals, that's lacking. Also, there should **not** be a bottom border nor a background on the totals row. (see design below) Scherm­afbeelding 2024-07-16 om 14 37 10
- [x] ~On /input it looks like the width of the paragraph of text "Klopt de naam van het stembureau met de naam op je papieren proces verbaal? Dan kan je beginnen." is defined as a percentage of the parent column. Please use max-width instead, to  allow the text to occupy more space on smaller viewports. (applies to all paragraph texts: use max-width instead of percentage) Scherm­afbeelding 2024-06-26 om 08 21 10~
- [x] All pages have 'Abacus' as page title. Pages should have descriptive titles
  - /user/login --> Inloggen - Abacus
  - /user/account/setup --> Account instellen  - Abacus
  - /overview --> Overzicht verkiezingen - Abacus
  - /input --> Kies een stembureau - Abacus
  - /input/030 --> Invoeren `<stembureau nummer>` `<stembureau naam>`  - Abacus
- [x] When navigating to another page, user should be scrolled to top of page (repro: make your window less high, open list of polling stations on '/input', scroll to bottom of page and pick the last polling station. You will end up on a scrolled down version of the data entry screen (tested on https://6b2ce232.urn-uitslag.pages.dev/input/030 with long list of political groups)
- [x] Toggle between /input/030/numbers and /input/030/recount: the navigation items below "Is er herteld?" move up and down by 1 or 2 pixels. (more or less the same seems to happen if you click from an item halfway down the list to the bottom item and back. All items between the item halfway and the bottom most move a couple of pixels)
  - Looked into this a bit more: looks like the selected menu item just has 1px more padding (top and bottom / or a border maybe) then the non selected items. This make everything shift a bit, which stands out more if the items you toggle between are not adjacent to each other
- [x] The clickable area for items in the navigation menu in /input/030/ consists of just the text of the link. Ideally the clickable target is a big as the box we highlight for selected menu-items
- [x] The selected menu-item is a link. This should not be the case, do not link to the current page.
- [x] In the nav-menu and in the h2, the name of the political group should be prepended with `Lijst <lijstnummer> - ` 
- [ ] There is no fallback for the Geist Mono font in the input screen. When loading this page for the first time, the user might run into a situation where the input fields are rendered with Times New Roman as font. Possible solutions: → #267
  - Define a DMSans as fallback for the GeistMono font for the `--font-number` var
  - Only render the page if all resources have been loaded
- [x] The tooltip on paste only works on `/input/:pollingstation-id/numbers`, and not `/input/:pollingstation-id/list/:listnumber`. It should be implementend on all numeric inputs
- [x] when a list has exactly n\*25 candidates, the spacer/separator for the next 25 candidates is shown. The separator should only be added if there is a (n\*25)+1'nd  candidate.  Scherm­afbeelding 2024-08-01 om 11 13 17
- [x] The zebra-striping of the rows does not match the striping used in the paper forms. On paper, the first row of each 25 rows is white. In our digital form, row 26 and row 76 have a dark background. Each block of 25 rows should start with a white background.
Scherm­afbeelding 2024-08-05 om 09 49 33
- [x] styling of the "Invoer afbreken" button isn't as designed (should not have an icon, padding / line-height looks off) → Fixed in #256 
- [x] Add hover state in navigation menu on ` /1/input/33/...` (see [Figma](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=123-2660&t=3sUgnTRRixRyz5ld-11))
- [x] Check colors of icons (current page, empty input) in the nav menu. Have been updated in Figma, they were a subtly different color then the texts, is now consistent with text
- [x] font-weights: het is ok om de 500 en 600 te vervangen door 700 (gebeurt op een aantal buttons al, daar wordt een bold over de 500 weight gezet, waardoor die 500 al niks meer doet). Nb: alleen voor de DM Sans teksten. De 500 op Space Grotesk (de badges) moet zo blijven
- [ ] Fix layout issues on "Alle stembureaus zijn twee keer ingevoerd" (margin above button) and make not closable → #267
- [ ] Only show "account is ingesteld" when account has just been set up → #267

### PR DESCRIPTION
**Pull request by praseodym**
Fixes #101
Fixes part of #263 

- Show account setup alert only after setup page
  - Move "Je account is ingesteld" to overview page (@jorisleker also see my comment in Figma)
  - Only show alert after account setup page
  - Remove close button from "Alle stembureaus zijn twee keer ingevoerd"
  alert
- Fix spacing on alerts
  - Fixes margin on "Alle stembureaus zijn twee keer ingevoerd"
alert button
- Fix styling of medium size button
  - Fixes padding of "Alle stembureaus zijn twee keer ingevoerd" alert
button
- Keep overview page empty while loading
- Set fallback font for data entry input fields
- Use number font on polling station number input

### CODE REVIEW

**Comment by jorisleker** [comment]
@praseodym how to test this: 'Only show alert after account setup page'? When I open http://localhost:8080/, it automatically redirects me to /overview, without showing account creating / login screens

**Comment by praseodym** [comment]
> @praseodym how to test this: 'Only show alert after account setup page'? When I open http://localhost:8080/, it automatically redirects me to /overview, without showing account creating / login screens.

In #259, the development screen was moved to [`/dev`](https://101-fix-design-debt.kiesraad-abacus.pages.dev/dev). The user login screen is still at [`/user/login`](https://101-fix-design-debt.kiesraad-abacus.pages.dev/user/login).

> Also: what curl command should I run to finalize my input and see the "Alle stembureaus zijn twee keer ingevoerd" alert?

`curl -v -XPOST http://127.0.0.1:8080/api/polling_stations/1/data_entries/1/finalise` after submitting a valid data entry form (all zeroes is valid).

File: frontend/lib/ui/InputGrid/InputGrid.module.css [hunk 1]
```diff
@@ -46,7 +46,7 @@
         border-right: 1px solid var(--bg-gray-darkest);
         padding: 0;
         input {
-          font-family: var(--font-number);
+          font-family: var(--font-number) var(--font-text);
```
Comment (jorisleker) [inline frontend/lib/ui/InputGrid/InputGrid.module.css:49]: @praseodym Wouldn't it be a better idea to set the fallback font in the `--font-number` variable? That way, we can define a fallback that is monospaced, just like the Geist Mono font, and retain as much of the original layout/design choices a possible. 

if we want to define a generic 'monospaced' fallback, we should do it like this : https://stackoverflow.com/questions/38781089/font-family-monospace-monospace

**Review by jorisleker — CHANGES_REQUESTED** [review]

File: frontend/lib/ui/InputGrid/InputGrid.module.css (hunk 1 above)
Comment (praseodym) [reply frontend/lib/ui/InputGrid/InputGrid.module.css]: Fixed in 4cd653fd1cf344c4f6d510df50fe6b21df634e74.

**Review by jorisleker — APPROVED** [review]

**Pull request merged**